templates/php-api/src/Processor/DebugProcessor.php

```
@@ -0,0 +1,32 @@
+    public function __construct(private readonly EntityManagerInterface $entityManager)
+    {
+    }
+
+    public function process(mixed $data, Operation $operation, array $uriVariables = [], array $context = [])
+    {
+        try {
+            $this->entityManager->remove($data);
+            $this->entityManager->flush();
+        } catch (Exception $e) {
+            if (method_exists($e, 'getCycle')) {
+                $cycleNodes = $e->getCycle();
```

PHPMD (2.15.0)

Avoid unused local variables such as '$cycleNodes'. (Unused Code Rules)

(UnusedLocalVariable)